gfootball/env/observation_processor.py

```diff
@@ -103,33 +143,35 @@ def get_frame(trace):
       field_coords=True,
       color=(255, 0, 0))
   writer.write('B')
-  for player_idx, player_coord in enumerate(trace['left_team']):
+  for idx, player_coord in enumerate(trace['left_team']):
+    player_idx = string.ascii_lowercase[idx]
     writer = TextWriter(
         frame,
         player_coord[0],
         player_coord[1],
         field_coords=True,
         color=(0, 255, 0))
-    letter = 'H'
-    if 'active' in trace and player_idx in trace['active']:
+    letter = 'W'
```

Comment:
Why W?
Can we number controlled players 0..9 instead of a,b...?

Reply:
'W' was an arbitrary choice, just didn't want to introduce confusion if one controlled player was named 'H'. 

I can change it back to 0..9, but I think a,b... notation can be clearer. Players controlled on the left team are numbered with lowercase a,b,... whilst on the right team uppercase A,B,... so it's easy to see to which team a player belongs. If we name players 0..9 there is a need to add an additional column in the table that says if player is in left/right team that can be less readable.

Reply:
Why can't we use color for that instead of additional column?

Reply:
That's a good point. I reverted the changes so there are used original letters e.g. 'H', 'A' and players are numbered 0..9. The player's team is denoted by its color in the table. Players in the table are sorted by:
1. team they are in
2. their number

Reply:
Thanks. How about one more change - what if we always number players 0..9 (and G for goal keeper), so that from the video it is clear which player is which.